scripts/scripts.js

```diff
@@ -44,6 +45,62 @@ async function loadFonts() {
   }
 }
 
+/**
```

Comment:
As this is part of the header logic, could you move it to "header.js"? As a note, anything in scripts.js (or most files for that matter) should be encapsulated in a function (not just called when scripts.js is included).